### ISSUE TITLE
Accessibility: SplitButton example should have an aria-label for the custom example (primary button)

### ISSUE THREAD

**Issue by kenotron** [OfficeDev/office-ui-fabric-react#10495]
### Environment Information

- **Package version(s)**: 7.x

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Splitting out #6356 to smaller bugs to be fixed. SplitButton example should have an aria-label for custom example (primary button)

### Repro

1. Open 'https://developer.microsoft.com/en-us/fabric#/components/button' in Edge.   
2. Navigate to the 'Split button with icon and custom styles' section under 'Implementation examples'.    
3. Open developer tool(F12 key).   
4. Observe the issue.

### Actual

Tooltip is not provided for 'Split button with icon' button.

### Expected

Tooltip should be provided for 'Split button with icon' button.

**Comment by kenotron** [comment]
Actually this is no repro anymore, I'm closing this one - #10494 remains to be the only issue
